### PR TITLE
Upgrade to netcoreapp2.2 and remove netcoreapp2.0 testing

### PR DESCRIPTION
**Pull request by BrennanConroy**
cc @ryanbrandenburg

### CODE REVIEW

**Comment by ryanbrandenburg** [comment]
Oh, NM, SignalR wasn't part of my script because it didn't exist the last time I did this.

**Comment by davidfowl** [comment]
Fix the commit title

File: benchmarkapps/BenchmarkServer/BenchmarkServer.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (BrennanConroy) [inline benchmarkapps/BenchmarkServer/BenchmarkServer.csproj]: @sebastienros I'm guessing benchmarks wont work with this yet?

**Comment by davidfowl** [comment]
I fixed it for you...

**Pull request merged**